.github/workflows/continous-integration.yml

```diff
@@ -254,7 +259,6 @@ jobs:
       if: needs.changes.outputs.backend == 'true'
       run: |
         echo "POETRY_VERSION=$(scripts/poetry-version.sh)" >> $GITHUB_ENV
-      shell: bash
```

Comment:
Why is this removed?

Reply:
[The job-level default shell is bash](https://github.com/RasaHQ/rasa/blob/b0fa94243abda1aa9504b2eacb5044a35a89575e/.github/workflows/continous-integration.yml#L220-L222). No difference really, just a preference of setting shell.

Reply:
[The job-level default shell is bash](https://github.com/RasaHQ/rasa/blob/b0fa94243abda1aa9504b2eacb5044a35a89575e/.github/workflows/continous-integration.yml#L220-L222). No difference really, just a preference of setting shell.